step_by_step_contribute.md

```diff
@@ -49,21 +49,21 @@ Please perform the following steps to create your Pull Request to this repositor
 
 2. Setting up test environment. See [Test Preparations](tests/README.md#Preparations) for more details.
 
-### Step 4: Pick up a challenge program
+### Step 4: Pick up a issue
 
 You can start by finding an existing issue with the
-[UCP](https://github.com/pingcap/dm/labels/challenge-program-2)
+[help wanted](https://github.com/pingcap/dm/labels/help%20wanted)
```

Comment:
If you want, you can add this label to UCP issues.

Reply:
Done.